app.rb

```diff
@@ -0,0 +1,154 @@
+require_relative 'book'
+require_relative 'classroom'
+require_relative 'person'
+require_relative 'rental'
+require_relative 'student'
+require_relative 'teacher'
+class App
+  def initialize
+    @books = []
+    @people = []
+  end
+
+  def run
+    puts 'Welcome to School Library App!'
+    list_options
+  end
+
+  def list_options
+    loop do
+      puts 'Please choose an option by entering a number:'
+      puts '1 - List all books'
+      puts '2 - List all people'
+      puts '3 - Create a person'
+      puts '4 - Create a book'
+      puts '5 - Create a rental'
+      puts '6 - List all rentals for a given person id'
+      puts '7 - Exit'
+
+      number = gets.chomp.to_i
+      case number
+      when 1..6 then execute_option(number)
+      when 7 then puts 'Thank you for using this app!'
+      end
+      break if number == 7
+    end
+  end
+
+  def execute_option(response)
+    case response
+    when 1
+      list_all_books
+    when 2
+      list_all_people
+    when 3
+      create_person_option
+    when 4
+      create_book_option
+    when 5
+      create_rental
+    when 6
+      rental_option
+    end
+  end
```

Comment:
- [ ] @manzitresor Great work implementing the change requested, as the previous reviewer had requested kindly consider moving this section to your main.rb file as required in the project description.
![image](https://github.com/manzitresor/school_library/assets/61408860/4c5bad03-003e-47e7-b0c4-c2a4dccb6af9)